__tests__/fixtures/fakeContentCreator.js

```diff
@@ -1,8 +1,10 @@
-const { encrypt } = require('../../helpers/password');
 const mongoose = require('mongoose');
 
-module.exports = function makeFakeContentCreator(baseUserId = mongoose.Types.ObjectId()) {
+module.exports = function makeFakeContentCreator(baseUserId, approved, rejected) {
     return {
+        //Created like this as to easily dictate their values for the purpose of the individual tests
+        approved: approved,
+        rejected: rejected,
```

Comment:
We did discuss this, but felt that having two different fields would allow for more flexibility in determining what and how for other aspects of the application. We are fine with either, but feel there has to be a clear distinction between rejected and approved, which translates well as two separate values.